packages/web3-providers-ws/test/unit/request.test.ts

```diff
@@ -1,38 +1,38 @@
-import { RpcOptions } from 'web3-providers-base/lib/types';
-import { EventEmitter } from 'events';
-
 import Web3ProviderWS from '../../src/index';
-import { WebSocketOptions } from '../../src/types';
-import { doesNotMatch } from 'assert/strict';
 import {
     Eth1RequestArguments,
+    RpcResponse,
     Web3ProviderEvents,
 } from 'web3-core-types/lib/types';
 
-describe('Web3ProvidersWS.request', () => {
-    const request: Eth1RequestArguments = {
-        rpcOptions: { id: 1, jsonrpc: '2.0' },
-        method: 'eth_blockNumber',
-        params: [],
-    };
-    jest.setTimeout(30000);
+describe.skip('Web3ProvidersWS.request', () => {
```

Comment:
These are unit tests, so the return values for the web socket client should be mocked

CI and an actual WS endpoint will be introduced via integration or e2e tests

Reply:
This PR is init tests only so I skipped it, based on same assumption you shared here.